aips/passkeys.md

```diff
@@ -0,0 +1,504 @@
+    /// key type specification, i.e. REQUIRED for the key type "kty" and algorithm "alg"
+    /// (see Section 2 of [RFC9053]).
+    ///
+    /// [RFC9052]: https://www.rfc-editor.org/rfc/rfc9052
+    /// [RFC9053]: https://www.rfc-editor.org/rfc/rfc9053
+    pub key: CoseKey,
+}
+```
+
+The credential public key is stored in the [CBOR Object Signing and Encryption format (COSE)](https://datatracker.ietf.org/doc/html/rfc8152) format. The `CoseKey` includes the `x` and `y` coordinates of the public key, which when combined, produces the public key. 
+
+With the raw public key, an Aptos account address can be derived. Note that the default authentication key derivation will vary depending on the account type created (`SingleKey` or `MultiKey`).
+
+### Signing a transaction
+
+In a traditional client-server implementation of the WebAuthn specification, both registering a new credential and authentication assertion use [challenge-response authentication](https://en.wikipedia.org/wiki/Challenge%E2%80%93response_authentication) to avoid several types of attacks. Randomized challenges are particularly important for protecting against [replay attacks](https://en.wikipedia.org/wiki/Replay_attack) [(§13.4.3)](https://www.w3.org/TR/webauthn-3/#sctn-cryptographic-challenges).
+
+Aptos is adapting the WebAuthn specification to be used for on-chain accounts and transactions. As many already know, on-chain transactions include a sequence number to avoid replay attacks on Aptos. Thus, if transactions are used as the challenge in a WebAuthn assertion, the replay attack risk is mitigated even if the challenge is not randomly generated by the relying party.
+
+#### [`challenge`](https://www.w3.org/TR/webauthn-3/#authenticatorassertionresponse)
+
+```js [code reference](https://www.w3.org/TR/webauthn-3/#dictdef-publickeycredentialrequestoptions)
+dictionary PublicKeyCredentialRequestOptions {
+    required BufferSource                   challenge;
+    unsigned long                           timeout;
+    USVString                               rpId;
+    sequence<PublicKeyCredentialDescriptor> allowCredentials = [];
+    DOMString                               userVerification = "preferred";
+    sequence<DOMString>                     hints = [];
+    DOMString                               attestation = "none";
+    sequence<DOMString>                     attestationFormats = [];
+    AuthenticationExtensionsClientInputs    extensions;
+};
+```
+
+During an assertion, the `SHA3-256` digest of `signing_message(raw_transaction)` is provided as the `challenge` in [`PublicKeyCredentialRequestOptions`](https://www.w3.org/TR/webauthn-3/#dictdef-publickeycredentialrequestoptions). The [`signing_message`](https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos-crypto/src/traits.rs#L151) function is applied before computing the `SHA3-256` of the `raw_transaction` to acheive domain separation without further increasing the size of the challenge. 
```

Comment:
It's good that you explain that passkeys are used to sign over a challenge and we simply replace that challenge with a non-replayable TXNs.

Again, not sure that you need to explain in low level all of the details.